src/content/pages/account/login/index.tsx

```diff
@@ -111,12 +118,43 @@ const LoginForm = () => {
 
         // Don't continue to login logic if there are errors.
         // We cannot check the useState values, as these will not be updated until the next render.
-        if (usernameInvalid || passwordInvalid) return
+        //if (usernameInvalid || passwordInvalid) return
 
         // TODO: Login logic.
 
+        const asyncPostCall = async () => {
+            try {
+                const response = await fetch('http://localhost:8000/user/login', {
+                 method: 'POST',
+                 headers: {
+                   'Content-Type': 'application/json',
+                   },
+                body: JSON.stringify({
+                    email: username,
+                    password: password
+                    })
+                 });
+                 const data = await response.json();
+                 setJWT(data)
+               } 
+               
+               catch(error) {
+                  console.log(error)
+                } 
+            }
+            
+        asyncPostCall()
```

Comment:
React components shouldn't have side effects when rendering. This should be called in a useEffect.